stdlib/2and3/filecmp.pyi

```diff
@@ -0,0 +1,49 @@
+# Stubs for filecmp (Python 2/3)
+import sys
+from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple
+
+_SupportsBool = object
```

Comment:
I made it a `Union[int, bool]` since that's how people should probably call it with.